docs-src/http-api.md

```diff
@@ -0,0 +1,530 @@
+  }
+}' -H "Content-Type: application/json"
+
+{
+  "docId": "kjzl6cwe1jw14ahmwunhk9yjwawac12tb52j1uj3b9a57eohmhycec8778p3syv",
+  "state": {
+    "doctype": "tile",
+    "content": {
+      "title": "My first Document"
+    },
+    "metadata": {
+      "schema": null,
+      "controllers": [
+        "did:key:z6MkfZ6S4NVVTEuts8o5xFzRMR8eC6Y1bngoBQNnXiCvhH8H"
+      ]
+    },
+    "signature": 2,
+    "anchorStatus": "PENDING",
+    "log": [
+      {
+        "cid": "bagcqcera2faj5vik2giftqxftbngfndkci7x4z5vp3psrf4flcptgkz5xztq",
+        "type": 0
+      },
+      {
+        "cid": "bagcqcera3fkje7je4lvctkam4fvi675avtcuqgrv7dn6aoqljd5lebpl7rfq",
+        "type": 1
+      }
+    ],
+    "anchorScheduledFor": "12/30/2020, 1:15:00 PM",
+    "next": {
+      "content": {
+        "title": "My first Document",
+        "more": 234
+      },
+      "metadata": {
+        "schema": null,
+        "controllers": []
+      }
+    }
+  }
+}
+
+
+```
+
+
+
+# Pins
+
+The pins api endpoint can be used to manipulate the pin set. The pin set is all of the documents that a node maintains the state of. Any document opened by the node that is not pinned will eventually be garbage collected from the node.
+
+## Add a document to the pin set
+This method adds the document with the given [[DocID]] to the pin set. 
+
+*`⚠️ This method is not available if gateway mode is enabled.`*
+
+### Request
+`POST /api/v0/pins/:docid`
+
+Here, `:docid` should be replaced by the string representation of the DocID of the document that is being requested.
+
+### Response
+
+* `docId` - the [[DocID]] of the document which was pinned, string
+* `isPinned` - whether the document was pinned, boolean - `true`
+
+### Example
+
+```bash
+curl http://localhost:7007/api/v0/pins/k2t6wyfsu4pg2qvoorchoj23e8hf3eiis4w7bucllxkmlk91sjgluuag5syphl -X POST
+
+{
+  "docId": "k2t6wyfsu4pg2qvoorchoj23e8hf3eiis4w7bucllxkmlk91sjgluuag5syphl",
+  "isPinned": true
+}
+```
+
+## Remove a document from the pin set
+This method removes the document with the given [[DocID]] from the pin set.
+
+*`⚠️ This method is not available if gateway mode is enabled.`*
+
+### Request
+`DELETE /api/v0/pins/:docid`
+
+Here, `:docid` should be replaced by the string representation of the DocID of the document that is being requested.
+
+### Response
+
+* `docId` - the [[DocID]] of the document which was unpinned, string
+* `isPinned` - whether the document was pinned, boolean - `false`
+
+### Example
+
+```bash
+curl http://localhost:7007/api/v0/pins/k2t6wyfsu4pg2qvoorchoj23e8hf3eiis4w7bucllxkmlk91sjgluuag5syphl -X DELETE
+
+{
+  "docId": "k2t6wyfsu4pg2qvoorchoj23e8hf3eiis4w7bucllxkmlk91sjgluuag5syphl",
+  "isPinned": false
+}
+```
+
+## Check if a document is in the pin set
+This method is used to check if a particular document is in the pin set.
+
+### Request
```

Comment:
a nitpick, but I find the flow of this document easier to read if the "list all documents in the pin set" section comes before the "check if document is in the pin set" section

Reply:
good point.